MODULE.bazel

```diff
@@ -0,0 +1,26 @@
+module(
+    name = "swiftlint",
+    version = "0.50.3",
+    compatibility_level = 1,
+    repo_name = "SwiftLint",
+)
+
+bazel_dep(name = "platforms", version = "0.0.5")
+bazel_dep(name = "rules_apple", version = "2.0.0", repo_name = "build_bazel_rules_apple")
+bazel_dep(name = "rules_swift", version = "1.5.1", repo_name = "build_bazel_rules_swift")
+bazel_dep(name = "rules_xcodeproj", version = "0.12.2", repo_name = "com_github_buildbuddy_io_rules_xcodeproj")
```

Comment:
Can this be marked as a dev dependency?

Reply:
AFAICT no because it's used in the root BUILD file, if it was in one that wasn't evaluated by downstream consumers then you could